app/views/unattended/report_templates/jobs_-_invocation_report_template.erb

```diff
@@ -30,15 +30,11 @@ require:
 <%- load_hosts(search: search).each do |batch| -%>
 <%-   batch.each do |host|  -%>
 <%-     task = invocation.sub_task_for_host(host) -%>
+<%-     outputs = { 'stdout' => [], 'stderr' => [], 'debug' => [] }  -%>
 <%-     task.action_continuous_output.each do |output| -%>
-<%-       report_row(
-            host: host.name,
-            result: task.result,
```

Comment:
This column is missing in the new response

Reply:
I think this is now resolved